HC 967 update trademark articles and categories (#43)
arduino/help-center-content: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md

---
title: "Arduino and Community logos"
---

This article covers the basic trademark rules that must be followed when using the [Arduino Logo](#arduino-logo) and the [Community Logo](#community-logo).

---

<h2 id="arduino-logo">Arduino Logo</h2>

The Arduino logo can be found in the following colors:

![Black Arduino Infinity Logo](img/ArduinoLogo_Black.jpg) ![Blue Arduino Infinity Logo](img/ArduinoLogo_Blue.jpg)

 > The logo can sometimes be accompanied by the word mark *Arduino* or appear on its own.

The Arduino logo is a registered trademark and cannot be used by third parties without the trademark owner’s authorization. Generally speaking, you **cannot** use the Arduino Logo (with or without the word *Arduino*) to identify or advertise your products and/or services. This includes:

* Compatible products <!-- link to Compatible products section-->
* Publications <!-- link to Courses, workshops and publications section-->
* Online content.

> > Exception when the Arduino Logo is used in a descriptive or explanatory way. For example, the Arduino Logo may be used within a text explaining what Arduino is.

Reviewer: Grammar doesn't seem quite right

Author: See [Update grammar](https://github.com/arduino/help-center-content/pull/43/commits/4566d7f834ccdf5f8cc83dda5d59f581d3a90a45)

Reviewer: > do you mean line 23?
Yes

> See [4566d7f](https://github.com/arduino/help-center-content/pull/43/commits/4566d7f834ccdf5f8cc83dda5d59f581d3a90a45)
This works for me!

Please be aware that even modified/customized versions of the Arduino Logo are considered trademark violations.

* [See examples](#examples) of correct and incorrect uses of Arduino logos. Note that the examples are for illustrative purposes.

<h2 id="community-logo">Community Logo</h2>

> The Arduino Community Logo ([download here](https://www.arduino.cc/en/Trademark/CommunityLogo)) has been developed by Arduino to give users the chance to identify non commercial pages, groups, events and initiatives connected to Arduino.

Reviewer: ```suggestion
The Arduino Community Logo ([download here](https://www.arduino.cc/en/Trademark/CommunityLogo)) has been developed by Arduino for users to identify non-commercial pages, groups, events, and initiatives related to Arduino.
```
For me, "connected" seems to imply some official endorsement, which I don't think is the case.

> ![Infinity Arduino Community Logo](img/ArduinoCommunityLogo.png)

The Arduino Community Logo can be used freely on, for example:

> * Social media community pages and groups, and to identify not for profit activities such as free events

Reviewer: ```suggestion
* Social media community pages and groups, and to identify not-for-profit activities such as free events
```
An event doesn't have to be free for it to be not-for-profit? Could be clarified.

Author: See [3443244](https://github.com/arduino/help-center-content/pull/43/commits/34432441d514c8be223f7a746b121920c6356980) Maybe we don't need to give examples of non-for-profit activities.

Reviewer: @Renat0Ribeir0 Yeah, that makes sense.

* Non commercial publications, courses and workshops. (e.g. when developing a not for profit video tutorial about Arduino, the Community Logo can be used in connection to it)

The Arduino Community Logo **cannot** be used on:

* Products or services that are sold commercially

> * Electronic boards (also non commercial)

Reviewer: ```suggestion
* Electronic boards (including non-commercial)
```

> **Note**: the Arduino Community Logo is shared with Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0) license. It is designed to allow personalization in the lower-right part in order to provide a free space on which to add the name of your group or organization. Apart from that, the community logo should not be altered, cropped, shortened or modified in any way. It is not allowed to incorporate another logo, trademark or commercial company name into the Community Logo.

Reviewer: ```suggestion
**Note**: the Arduino Community Logo is shared with a [Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0) license](https://creativecommons.org/licenses/by-nc-sa/3.0/). It is designed to allow personalization in the lower-right part in order to provide a free space on which to add the name of your group or organization. Apart from that, the community logo should not be altered, cropped, shortened, or modified in any way. It is not allowed to incorporate another logo, trademark, or commercial company name into the Community Logo.
```
